### PR TITLE
LPS-101720 Add Email One-Time password checker

### PR DESCRIPTION
**Pull request by csierra**
/cc @arthurchan35

### CODE REVIEW

**Comment by brianchandotcom** [comment]
@csierra can we do a quick call tomorrow over this? I want to do a screen share with you so we can get this feature merged in.. right now.. the velocity is so slow because there's a fundamental misunderstanding in how things are named... but once you get the scheme.. everything moves very fast.. but I don't want to merge in code that doesn't fit the existing naming scheme. I can do it earlier before our calls or right after our calls.

In short:

MultiFactorAuthenticationConfigurationCategory

MFAEmailOTP vs. multi-factor-authentication-checker-email-otp-web (see the naming Checker is missing)

I don't mean to drive you mad.. but the naming in this module is driving me mad... so let's talk tomorrow so we can find a way to get your velocity fast again (it is possible.. I merge 50+ pulls a day with many many lines of code).

**Comment by csierra** [comment]
hey @brianchandotcom,
sure thing... let me know when it suits you best.
Thx.
Carlos.

**Comment by brianchandotcom** [comment]
@csierra will resend.
